simple-menu.php

```diff
@@ -1,20 +1,26 @@
 <?php
-/*
-Plugin Name: Genesis Simple Menus
-Plugin URI: https://github.com/copyblogger/genesis-simple-menus
+/**
+ * Plugin Name: Genesis Simple Menus
+ * Plugin URI: https://github.com/copyblogger/genesis-simple-menus
+ * Description: Genesis Simple Menus allows you to select a WordPress menu for secondary navigation on individual posts/pages.
```

Comment:
Do we want to include taxonomies in the description? It also works on categories and tags.

Reply:
It would be nice to add that @dreamwhisper IMHO.
Something like:
```
* Description: Genesis Simple Menus allows you to select a WordPress menu for secondary navigation on individual posts, pages, taxonomies, categories, and tags.
```
LMK if that works and I'd be happy to change that :)

Reply:
@marksabbath I think we can just say taxonomies as that's inclusive of categories and tags.

> * Description: Genesis Simple Menus allows you to select a WordPress menu for secondary navigation on individual posts, pages, and taxonomies.

Reply:
Ok, sounds good to me. I've pushed another commit and should be all resolved now :)